poetry/utils/_compat.py

```diff
@@ -263,26 +263,6 @@ def encode(string, encodings=None):
     return string.encode(encodings[0], errors="ignore")
 
 
-def to_str(string):
-    if isinstance(string, str) or not isinstance(string, (unicode, bytes)):
-        return string
-
-    if PY2:
-        method = "encode"
-    else:
-        method = "decode"
-
-    encodings = ["utf-8", "latin1", "ascii"]
```

Comment:
Should we handle this via `str(x, encoding=encoding)`? We should add a test case for PyPI repositories that evalueatets this.